### PR TITLE
Added range sensor costmap layer to documentation

### PR DESCRIPTION
**Pull request by Michael-Equi**
Connected to https://github.com/ros-planning/navigation2/pull/1888

### CODE REVIEW

File: configuration/packages/costmap-plugins/range.rst [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+
+  Description
+    Whether to clear the sensor readings on max range.
+
+:``<range layer>``.input_sensor_type:
+
+  ====== =======
+  Type   Default
+  ------ -------
+  int    ALL
```
Comment (SteveMacenski) [inline configuration/packages/costmap-plugins/range.rst:103]: Isn't this a string input?

File: configuration/packages/costmap-plugins/range.rst [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+
+  Description
+    Probability above which cells are marked as occupied.
+
+:``<range layer>``.clear_on_max_reading:
+
+  ====== =======
+  Type   Default
+  ------ -------
+  int    False
```
Comment (SteveMacenski) [inline configuration/packages/costmap-plugins/range.rst:92]: int != bool

File: configuration/packages/costmap-plugins/range.rst [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+
+  Description
+     Probability below which cells are marked as free.
+
+:``<range layer>``.mark_threshold:
+
+  ====== =======
+  Type   Default
+  ------ -------
+  int    0.8
```
Comment (SteveMacenski) [inline configuration/packages/costmap-plugins/range.rst:81]: int != 0.8

File: configuration/packages/costmap-plugins/range.rst [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+
+  Description
+    Phi value.
+
+:``<range layer>``.inflate_cone:
+
+  ====== =======
+  Type   Default
+  ------ -------
+  int    1.0
```
Comment (SteveMacenski) [inline configuration/packages/costmap-plugins/range.rst:46]: int != 1.0

File: configuration/packages/costmap-plugins/range.rst [hunk 5]
```diff
@@ -0,0 +1,107 @@
...
+
+  Description
+    Whether it is enabled.
+
+:``<range layer>``.topics:
+
+  ==== =======
+  Type Default
+  ---- -------
+  bool [""]
```
Comment (SteveMacenski) [inline configuration/packages/costmap-plugins/range.rst:24]: vector of strings?

**Comment by Michael-Equi** [comment]
> * migration guide

Which migration guide would you like this in? Eloquent -> Foxy or Foxy -> Galactic?

**Comment by SteveMacenski** [comment]
E to F, we dont change API so I can cherry pick this to Foxy. Its on the list https://github.com/ros-planning/navigation2/issues/1866 for inclusion next sync

**Pull request merged**